fio-plot/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.7.8-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+ARG VERSION=3.12-2
+
+ADD . /
+
+WORKDIR /
+
+RUN apt update && \
+    apt install -y --no-install-recommends fio=${VERSION} && \
+    pip install -r requirements.txt && \
+    rm -rf /var/lib/apt/lists/* && \
+    ln -s /bench_fio /usr/local/bin/bench_fio && \
```

Comment:
since the commands are installed to `/usr/local/bin` and this folder is in `PATH` by default, they can be referenced directly, e.g.

```
docker run --rm -ti getpopper/fio-plot bench_fio --args --to --benchfio
```

so there's no need to symlinking them

Reply:
So, these are just the executable scripts in the `/` directory. They are not installed anywhere. I am symlinking them to binary to make it available as bash commands. Now that I am refactoring the `Dockerfile`, let me see how to achieve this.

Reply:
oh, sorry, yeah, I got them backwards. Didn't realize that you're actually creating the links so they're in /usr/local/bin. Symlinking would be fine, as the fio-plot project doesn't have a proper pip package

Reply:
Yeah. cool.